scripts/pack_docs.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+from subprocess import check_call
+import os
+
+def do_the_job(src, dst):
+    ''' Helper '''
+    def pack(files, res, cwd):
+        if len(files) < 1:
+            print("No items to pack in %s" % cwd)
+            return
+        print("")
+        print("====== Pack %d items in %s to %s" % (len(files), cwd, res))
+        print("")
+        check_call(["zip", "-r", "-9", "-y", os.path.join(dst, res)] + files, cwd=cwd)
+
+    doxy_path = os.path.join(src, "doxygen", "html")
+    if os.path.isdir(doxy_path):
+        pack(os.listdir(doxy_path), "doc_doxygen.zip", doxy_path)
+
+    sphinx_path = os.path.join(src, "_html")
+    if os.path.isdir(sphinx_path):
+        pack(os.listdir(sphinx_path), "doc_sphinx.zip", sphinx_path)
```

Comment:
Sphinx was used in 2.4. Most probably it could be safely dropped.

Reply:
Removed it.